packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -23,17 +23,23 @@ import PostPreviewButton from '../post-preview-button';
 import { store as editorStore } from '../../store';
 
 export default function PostLockedModal() {
+	const [ heartbeatTickTriggered, triggerHeartbeatTick ] = useState( false );
 	const instanceId = useInstanceId( PostLockedModal );
 	const hookName = 'core/editor/post-locked-modal-' + instanceId;
 	const { autosave, updatePostLock } = useDispatch( editorStore );
 	const {
+		__experimentalUpdateCurrentUserSession: updateCurrentUserSession,
+	} = useDispatch( coreStore );
+	const {
+		isDraft,
 		isLocked,
 		isTakeover,
 		user,
 		postId,
 		postLockUtils,
 		activePostLock,
 		postType,
+		isCurrentUserSessionActive,
```

Comment:
ℹ️  For now, this data (`isCurrentUserSessionActive`) is only being used in this component and could well be a local state instead of being in the data store. But I can think of few use cases where we would want to check for current user sessions. I myself thought about using the same in `AutosaveMonitor` component. (This was when I was exploring potential fix in this PR. Eventually the need did not arise. So I removed that code before committing.) But the point is, there is a use case and can be a need in future.